docs/docsite/rst/dev_guide/developing_inventory.rst

```diff
@@ -61,6 +61,40 @@ When using the 'persistent' cache, inventory plugins can also use the configured
 
 .. _developing_an_inventory_plugin:
 
+Directory layout
+----------------
+
+The `PluginLoader` class defines a property named `config`, which is a list of directories that will be searched for inventory plugins. By default, it may contain::
+
+   /home/foo/.ansible/plugins/inventory
+   /usr/share/ansible/plugins/inventory
+   /usr/lib/python3.7/site-packages/ansible/plugins/inventory
+   /usr/lib/python3.7/site-packages/ansible/plugins/inventory/__pycache__
+
```

Comment:
Giving a hard-coded list is misleading for the reader, because value can be are not correct, for instance `/usr/share/ansible/plugins/inventory` is I assume for ansible installed using system package, what about pip installation ? Same also for `/usr/lib/python3.7/site-packages/ansible/plugins/inventory` because it uses  `python3.7` is there a better way to list them using variables (`$PREFIX` ?) and possibly put one example after ?

Reply:
I completely agree. Sorry, I don't know of a better way to word it than "it may contain". PREFIX sounds good, though I'm a bit lost on how we'd use it here.

Reply:
No problem, that's what reviews are for, exchanging ideas.